skins/oasis/css/core/ads.scss

```diff
@@ -267,3 +267,19 @@ body >a >img[src$="noad.gif"] {
 		margin: 0;
 	}
 }
+
+// Recovered message
+.recovered-message {
+	background: #eee;
+	color: #000;
+	font-size: 2em;
+	text-align: center;
+	padding: 10px 0;
+
+	&.recovered-message-top {
+		margin: 0 auto;
+		position: relative;
+		width: 970px;
+		z-index: 2;
```

Comment:
We might want to use one of the [SASS variables](https://github.com/Wikia/app/blob/dev/skins/oasis/css/core/layout.scss#L2) but maybe it's a part of the other story. It's up to you.